sda-commons-server-kms-aws/src/test/java/org/sdase/commons/server/kms/aws/config/KmsAwsCachingConfigurationTest.java

```diff
@@ -0,0 +1,49 @@
+package org.sdase.commons.server.kms.aws.config;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class KmsAwsCachingConfigurationTest {
+  KmsAwsCachingConfiguration classUnderTest = new KmsAwsCachingConfiguration();
```

Comment:
Not sure I miss here something, but is `classUnderTest` a shared state between the tests? Are the tests order independent?